attributes/dotnet_agent.rb

```diff
@@ -3,6 +3,6 @@
 default['appdynamics']['dotnet_agent']['checksum'] = '7acb3756147a1d5a13c49b107a890ea56a8eb4099fd793e498e34b6f0b5962dc' #nil
 default['appdynamics']['dotnet_agent']['install_dir'] = 'C:\Program Files\Appdynamics'
 default['appdynamics']['dotnet_agent']['logfiles_dir'] = 'C:\DotNetAgent\Logs'
-
+default['appdynamics']['dotnet_agent']['standalone_apps'] = nil #'a.exe,b.exe'
```

Comment:
Does comma separated make sense here? Why not take an array?

```ruby
default['appdynamics']['dotnet_agent']['standalone_apps'] = ['a.exe', 'b.exe']
```

Then you don't have to split in the template.